pep-0001.txt

```diff
@@ -458,19 +458,25 @@ if they are never meant to be completed.  E.g. :pep:`1` (this PEP).
 PEP Maintenance
 ---------------
 
-In general, Standards track PEPs are no longer modified after they have
-reached the Final state. Once a PEP has been completed, the Language and
-Standard Library References become the formal documentation of the expected
-behavior.
+In general, PEPs are no longer substantially modified after they have reached
+the Accepted, Final, Rejected or Superseded state. Once resolution is reached,
+a PEP is considered a historical document rather than a living specification.
+Formal documentation of the expected behavior should be maintained elsewhere,
+such as the `Language Reference`_ for core features, the `Library Reference`_
+for standard library modules or the `PyPA Specifications`_ for packaging.
 
 If changes based on implementation experience and user feedback are made to
-Standards track PEPs while in the Accepted or Provisional State, those changes
-should be noted in the PEP, such that the PEP accurately describes the state of
+Standards track PEPs while in the Provisional or (with SC approval) Accepted
+state, they should be noted in the PEP, such that the PEP accurately describes
 the implementation at the point where it is marked Final.
 
-Informational and Process PEPs may be updated over time to reflect changes
-to development practices and other details. The precise process followed in
-these cases will depend on the nature and purpose of the PEP being updated.
+Active Informational and Process PEPs may be updated over time to reflect
```

Comment:
```suggestion
Active, Informational and Process PEPs may be updated over time to reflect
```

Reply:
Actually, the intended syntax is `Active (Informational and Process) PEPs`, not `(Active, Informational and Process) PEPs` as the above change would imply. Any suggestions to make that clearer (as that obviously wasn't what came across)? E.g.

```suggestion
Active-status Informational and Process PEPs may be updated over time to reflect
```

or

```suggestion
Informational and Process PEPs that are Active may be updated over time to reflect
```

or even as a literal parenthetical?

```suggestion
Active (Informational and Process) PEPs may be updated over time to reflect
```

Reply:
The last one looks clearest to me.